plugin/evm/validators_default.go

```diff
@@ -0,0 +1,93 @@
+package evm
+
+import (
+	"fmt"
+	"math/big"
+	"os"
+	"strings"
+
+	"github.com/flare-foundation/coreth/params"
+	"github.com/flare-foundation/flare/ids"
+	"github.com/flare-foundation/flare/utils/constants"
```

Comment:
Agreed. I'm trying not to be too dogmatic in this code base, because otherwise we would change hundreds of files, and then we get more merge conflicts on upstream upgrades.